source/_integrations/smart_meter_b_route.markdown

```diff
@@ -0,0 +1,83 @@
+---
+title: Smart Meter B Route
+description: Integrate your Smart Meter using B Route.
+ha_category:
+  - Energy
+  - Sensor
+ha_release: '2025.10'
+ha_iot_class: Local Polling
+ha_codeowners:
+  - '@SeraphicRav'
+ha_domain: smart_meter_b_route
+ha_platforms:
+  - sensor
+ha_config_flow: true
+ha_integration_type: device
+---
+
+The **Smart Meter B Route** {% term integration %} allows you to access the data of your smart meter using the B Route protocol.
+
+## Overview
+
+This integration enables you to access and monitor your household energy consumption data from a smart meter using the B Route protocol in Home Assistant. You'll need a compatible USB dongle and your B Route credentials to set up this integration. It has been tested for the Japanese market and most households are eligible, the installation of a smart meter is free. You can find some information by TEPCO [here](https://www.tepco.co.jp/en/pg/development/domestic/smartmeter-e.html) but other utility companies may have similar pages.
+
+## Prerequisites
+
+Even if you have a contract with a new power provider, you can still use this service by applying to the regional utility company (for example, TEPCO in Tokyo).
+Ensure that your power supply is not a bulk power supply for an apartment building.
+
+1. **Compatible USB dongle**: Tested with the Wi-SUN USB dongle from [Ratoc Systems](https://www.ratocsystems.com/products/wisun/usb-wisun/rs-wsuha/).
+2. **B Route credentials**: Obtain your B Route credentials from your utility company.
+3. **Smart meter**: A smart meter that supports the B Route protocol. If you are renting but do not have a smart meter equipped, you will need to contact your house management company, as the installation is free but may require permission.
```

Comment:
```suggestion
1. Compatible USB dongle: 
    - Tested with the Wi-SUN USB dongle from [Ratoc Systems](https://www.ratocsystems.com/products/wisun/usb-wisun/rs-wsuha/).
2. [B Route credentials from your utility company](#optaining-b-route-credentials).
3. Smart meter: 
    - A smart meter that supports the B Route protocol. 
    - If you are renting but do not have a smart meter equipped, you will need to contact your house management company, as the installation is free but may require permission.
```
Because bold is used to markup UI strings, we avoid using it to highlight text, if possible.